tests/components/cast/test_config_flow.py

```diff
@@ -64,7 +65,7 @@ async def test_user_setup(hass):
     result = await hass.config_entries.flow.async_configure(result["flow_id"], {})
 
     users = await hass.auth.async_get_users()
-    assert len(users) == 1
```

Comment:
These tests didn't account for the content user which will also be created because we now await

Reply:
OK, but I think the purpose of this assert is a bit hard to understand.
Can it either be rewritten or have a comment added to it to make the purpose clearer?
```
assert next(user for user in users if user.name == CAST_USER_NAME)
```

Reply:
NVM this comment, @frenck confirmed I'm just being dense 🙈